specs/commandline/plugin.md

```diff
@@ -41,51 +43,151 @@ Aliases:
 ### notation plugin install
 
 ```text
-Installs a plugin
+Install a plugin
 
 Usage:
-  notation plugin install [flags] <plugin path>
+  notation plugin install [flags] <plugin_source>
 
 Flags:
   -h, --help                    help for install
   -f, --force                   force the installation of a plugin
-
+      --checksum string         must match SHA256 of the plugin source
```

Comment:
What's the format of `checksum`?

Reply:
It's the encoded value without `sha256:`.